dev/maven-plugin/src/main/maven/plugin.xml

```diff
@@ -409,6 +409,15 @@
                 <parameter>
                     <name>cassandraEnabled</name>
                 </parameter>
+                <parameter>
+                    <name>kafkaPort</name>
+                </parameter>
+                <parameter>
+                    <name>kafkaAddress</name>
+                </parameter>
```

Comment:
in sbt, `lagomKafkaAddress` is a property that holds a comma-separated list of brokers (https://github.com/lagom/lagom/blob/b91780456fb7f5017259fdc96b18c5307874de49/dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala#L310-L310). If we add a similar property to Maven I'd suggest the name `kafkaBrokerAddresses`.

We'll have to document that `kafkaPort` is only used the `kafkaBrokerAddresses` is not provided. (new PR)